src/style.js

```diff
@@ -120,10 +120,16 @@ export function setStyle(element, property, value, opt_units, opt_bypassCache) {
     property,
     opt_bypassCache
   );
-  if (propertyName) {
-    element.style[propertyName] = /** @type {string} */ (opt_units
-      ? value + opt_units
-      : value);
+  if (!propertyName) {
+    return;
+  }
+  const styleValue = /** @type {string} */ (opt_units
+    ? value + opt_units
+    : value);
+  if (isVar(propertyName)) {
+    element.style.setProperty(propertyName, styleValue);
```

Comment:
Can we just use this all the time and remove the `else` clause? [Browser support](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty) seems fine.

Reply:
Moved this to https://github.com/ampproject/amphtml/pull/27145

Reply:
I'm going to submit as is and will bring this up with you on Monday. There are some special cases there.